.github/ISSUE_TEMPLATE/doc-issue.md

```diff
@@ -5,27 +5,33 @@ about: Create an issue to help us improve
 
 # Before you open an issue
 
-If the issue is with an ASP.NET Core document, open the issue with the **Content feedback** button at the bottom of that document page.
+If the issue is related to an ASP.NET Core document, open the issue with the **This page** feedback button and form at the bottom of that document page. When you use the **This page** feedback form:
+
+* The issue automatically receives tracking metadata.
+* The author of that topic automatically receives a notification.
 
 If the issue is:
 
-- A simple typo or similar correction, you can submit a PR. See [the contributor guide](https://docs.microsoft.com/contribute/#quick-edits-to-existing-documents) for instructions.
-- A general support question, consider asking on a support forum site.
-- A site design concern, create an issue at [MicrosoftDocs/Feedback](https://github.com/MicrosoftDocs/Feedback/issues/new/choose).
-- A problem completing a tutorial, compare your code with the completed sample.
-- A duplicate of an open or closed issue, leave a comment on that issue.
+* A simple typo or similar correction, you can submit a pull request (PR). See [the contributor guide](https://docs.microsoft.com/contribute/#quick-edits-to-existing-documents) for instructions.
+* A general support or coding question, ask on a support channel, such as [Stack Overflow](https://stackoverflow.com/questions), [ASP.NET Core Slack](http://tattoocoder.com/aspnet-slack-sign-up/), or [ASP.NET Gitter](https://gitter.im/aspnet/Home). Our documentation workload usually prevents us from answering general requests for assistance.
+* A site design concern, create an issue at [MicrosoftDocs/Feedback](https://github.com/MicrosoftDocs/Feedback/issues/new/choose).
+* A problem completing a tutorial, compare your code with the completed sample.
+* A duplicate of an open issue, leave a comment on that issue.
+* A duplicate of a closed issue or related to a closed PR, open a new issue from the bottom of the document page and cross-reference the original issue or PR in the opening comment.
+
+*Please don't comment on closed issues and PRs. Cross-link the closed issue/PR on a new issue.*
```

Comment:
Strongly disagree on this one. There's valuable information in many closed issues. We should have much more emphasis on hiding closed issues, so only useful closed issues are tied to a topic. Only the 10 newest closed issues are displayed.

A closed issue with useful info should get fresh comments.